src/js/collect.jsx

```diff
@@ -29,35 +30,11 @@ import { visiblePanelAtom, extraMapParamsAtom, showModalAtom } from "./home";
 import CollectMap from "./collect/CollectMap";
 import NavBar from "./collect/NavBar";
 
-const BarItem = styled.div`
-  margin: 0 2rem;
-`;
-
-const Buttons = styled.div`
-  display: flex;
-  flex: 3;
-  justify-content: flex-start;
-`;
-
-const Logo = styled.div`
-  align-items: center;
-  display: flex;
-  flex: 1;
-  justify-content: space-evenly;
-  padding: 5px 0;
-`;
-
-const LogoGitVersion = styled.a`
-  color: var(--white);
-  cursor: pointer;
-  font-size: 12px;
-  letter-spacing: 0px;
-  text-align: left;
-  text-decoration: none;
-`;
-
 export const currentPlotIdAtom = atom(-1);
 export const currentPlotNumberAtom = atom(1);
+export const projectPlotsAtom = atom([])
+export const areAllPlotsValidatedAtom = atom((get) => get(projectPlotsAtom).every(p => p.answer))
```

Comment:
This might not be the right logic. When I go to a project where not all plots are validated I'm getting the popup telling me that all plots are validated.